### DISCUSSION TITLE
Features of the native tabs not covered by Tree Style Tab

### DISCUSSION THREAD

**Discussion by page200** [piroor/treestyletab#3449] [Q&A] [answered]
[The README.md file](https://github.com/piroor/treestyletab/blob/trunk/README.md) says:

> How to hide the top tab bar (horizontal tab strip)?</summary>
>
>  As a workaround, you can [create a `userChrome.css` file.](https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#for-userchromecss)
  But please remind that I - the original author of TST - never recommend such an usage, because TST doesn't cover full features of the native tabs due to restrictions of WebExtensions API so *some tab features become inaccessible*

What tab features are those? Thanks!

**Comment by piroor** [comment]
* Sharing state icon. It will be available at the next release of TST (maybe 4.0)
* "Autoplay blocked" message and the "Play tab" button ![image](https://github.com/piroor/treestyletab/assets/70062/122ee81f-3cd2-4e4a-a52a-3033a628fc44)
* "Share" tab context menu command
* "Send Tab to Device" tab context menu command. TST currently simulates it but you cannot send tabs to mobile devices.

**Comment by page200** [comment]
Please clarify this in `README.md` where the answer to "How to hide the top tab bar (horizontal tab strip)?" vaguely says "some tab features". Thanks!

**Reply by piroor** [reply, marked as answer]
Thank you for suggestion, I've done it!

**Reply by page200** [reply]
Thank you!
